test/e2e/testdata/cassandracluster-1DC-no-autopilot.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: jolokia-auth
+type: Opaque
+data:
+  password: TTBucDQ1NXcwcmQ=
+  username: am9sb2tpYS11c2Vy
```

Comment:
You can also get rid of the secret and remove it from your `CassandraCluster`. It's already tested elsewhere.

Reply:
@jsanda not sure that's you wanted to answer to that comment ...